### DISCUSSION TITLE
Welcome to MicaForEveryone Discussions!

### DISCUSSION THREAD

**Discussion by xmine64** [MicaForEveryone/MicaForEveryone#63] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dongle-the-gadget** [comment]
👋

**Comment by RobGrims** [comment]
Sorry to ask this as it must be obvious but this functionality is exactly what I'm chasing but I have no idea how to actually install it - couldn't find anything on the wiki, google searching, or the readme/files.  Apologies for the simple question!

**Reply by xmine64** [reply]
No need to install, just run it

**Comment by knowledgetechid** [comment]
This app package’s publisher certificate could not be verified. Contact your system administrator or the app developer to obtain a new app package with verified certificates. The root certificate and all immediate certificates of the signature in the app package must be verified (0x800B010A)

![image](https://user-images.githubusercontent.com/101148765/171764728-6276e72b-a82a-4056-8dad-de68dd89cfe1.png)

**Reply by xmine64** [reply]
Check wiki.

**Comment by dasai98** [comment]
Any tips on removing the program cleanly?

**Reply by xmine64** [reply]
if you installed it as msix, just uninstall it. if you don't it doesn't have anything other than the files that you extracted from the zip file, but it saves its config in the file that's set in general page of settings. also location of this config file is saved in registery.

**Comment by Laxdal** [comment]
i cant access settings, when ever i click the icon in system tray, a window pop up for 2senconds then disappreas. i tried with both versions, zip and msix

**Comment by Shorono** [comment]
This is probably a dumb question but where is the config file actually stored?

**Reply by dongle-the-gadget** [reply]
Should be in the File Path section of the Settings window.

**Reply by Shorono** [reply]
I may or may not have removed the icon from the toolbar and i can't find a way to force open the settings window as it seems to launch minimized.

**Reply by xmine64** [reply]
If you open MFE again while it's running it will open settings window.
